.travis.yml

```diff
@@ -0,0 +1,14 @@
+sudo: required
+dist: trusty
+language: go
+go:
+  - 1.5
+
+env:
+  matrix:
+    - TEST_SUITE="--static"
+    - TEST_SUITE="--unit-go"
+
+install:
```

Comment:
I prefer to make it clear why we don't have a install step. But anyway, this will go away when we do the installation of the node stuff.